src/utilities/stl_ops.c

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ * Copyright (c) 1998 Lawrence Livermore National Security, LLC and other
+ * HYPRE Project Developers. See the top-level COPYRIGHT file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ ******************************************************************************/
+
+#include "_hypre_onedpl.hpp"
+#include "_hypre_utilities.h"
+#include "_hypre_utilities.hpp"
+#include <math.h>
+
+/*--------------------------------------------------------------------
+ * hypre_IntSequence
```

Comment:
Can you add a description of what this function is doing?

Reply:
Done in [f6cfb03](https://github.com/hypre-space/hypre/pull/1102/commits/f6cfb03550cc398130830b521df0ffe4c3abd2c4)